model/storage/disk/utils.py

```diff
@@ -43,3 +44,31 @@ def remove_dir_out_of_grace(path: str, grace_period_seconds: int):
 
     if last_modified < datetime.datetime.now() - grace:
         shutil.rmtree(path=path, ignore_errors=True)
+
+
+def get_hash_of_file(path: str) -> str:
+    blocksize = 64 * 1024
+    sha = hashlib.sha256()
+    with open(path, "rb") as fp:
+        while True:
+            data = fp.read(blocksize)
+            if not data:
+                break
+            sha.update(data)
+    return sha.hexdigest()
```

Comment:
Can we use Base64 encoding so it only requires 44 bytes instead of 64?

Reply:
Done.